<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>6</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="ProductTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>Kategorie 1</string>
        </attribute>
        <widget class="QSpinBox" name="spinBox_Example">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>35</y>
           <width>51</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
        <widget class="AddProductButton" name="pushButton_Example">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>191</width>
           <height>71</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>51</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&lt;Leer&gt;</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_Example">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>51</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string notr="true">0,00â‚¬</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSlider" name="slider_Example">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>2</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Kategorie 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTableView" name="productTable">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>350</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>208</width>
         <height>913</height>
        </rect>
       </property>
       <widget class="AddProductButton" name="pushButton_Example_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>191</width>
          <height>71</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;Leer&gt;</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="AddProductButton" name="pushButton_Example_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>191</width>
          <height>71</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>51</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;Leer&gt;</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QComboBox" name="tableComboBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>5</y>
           <width>321</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxCount">
          <number>10</number>
         </property>
         <item>
          <property name="text">
           <string>Tischnummer 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tischnummer 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tischnummer 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tischnummer 4</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="prevTableButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>0</y>
           <width>34</width>
           <height>34</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>34</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="nextTableButton">
         <property name="geometry">
          <rect>
           <x>365</x>
           <y>0</y>
           <width>34</width>
           <height>34</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>34</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="bonTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTipDuration">
         <number>1</number>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::CopyAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QLineEdit" name="sumLineEdit">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>0</y>
           <width>141</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>0,00â‚¬</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>30</y>
           <width>141</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>58</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cashOutButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>60</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Abrechnen</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>0</y>
           <width>58</width>
           <height>21</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Summe:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="sortCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>86</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Sortieren</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddProductButton</class>
   <extends>QPushButton</extends>
   <header>addproductbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ProductTabWidget</class>
   <extends>QTabWidget</extends>
   <header>producttabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
